### ISSUE TITLE
Ability to receive a notification when a follower faves a shared post

### ISSUE THREAD

**Issue by atomicthumbs** [bluesky-social/social-app#6706]
### Describe the Feature

It'd be nice to know my friends liked a post I shared. There's no non-technical reason Bluesky shouldn't have this, since likes are public; when I find something and share it, but can't see whether anyone liked it, it feels like I'm interacting with a void.

**Comment by surfdude29** [comment]
Yeah this is quite a big difference with Twitter (especially for those of us who are heavy reposters), because on Bluesky you get no notification at all when someone likes or reposts a post that you reposted.

I think adding this to Bluesky would be good, although ideally with an option to turn off these notifications, as I'm guessing some people would not like them.

**Comment by ppoutine** [comment]
Yes. I had at first just assumed this worked like Twitter where you got a notification when someone liked your retweet. So I thought everyone hated my retweets for some reason. No, it's just a missing feature. These notifications were helpful feedback.

**Comment by surfdude29** [comment]
This was implemented a while ago :)
